modified Marr post

rawskijon opened the pull request.

thomas--graf commented:
I'm currently in Amsterdam, so it will probably take a few days before I merge this in. You can keep revising in the meantime if necessary.

rawskijon commented:
Sorry for the pull request mess, I didn't understand at first that I could just keep one draft pull open while updating my fork.

thomas--graf commented on content/Discussions/2019-12-31-Rawski_Marr.md:
```diff
@@ -0,0 +1,189 @@
+He places Chomsky's competence/performance distinction as the difference between the "computational theory" and "representation/algorithmic" levels.
+He even talks about trace theory a la Chomsky \& Lasnik showing "some of the rather ad hoc restrictions that form part of the computational theory may be consequences of weaknesses in the computational power that is available for implementing syntactical decoding".
+Linguists adapted this endorsement into the following argument (see above meme):
+
```
Is this actually a quote? If it isn't, typeset it differently. E.g.

```
(@folkmarr) **Linguists' take on Marr** (wow, that's one important apostrophe)  
Marr distinguishes...
```

thomas--graf commented on content/Discussions/2019-12-31-Rawski_Marr.md:
```diff
@@ -0,0 +1,189 @@
+Poggio notes that "one key aspect of the original argument disappeared in the process".
+What was it?
+"It is necessary to study nervous systems at all levels simultaneously".
+Great advice.  
```
There might be trailing spaces here, there's a sudden linebreak in the typeset article.

thomas--graf commented on content/Discussions/2019-12-31-Rawski_Marr.md:
```diff
@@ -0,0 +1,189 @@
+tags:
+    - neuroscience
+    - representations
+    - marr
```
`- Marr`

thomas--graf commented:
I spotted three very minor issues, check the comments in the code view. Once those are fixed, I'll merge it into `staging` and then push it out to `master` whenever you'd like (Sunday?). I might add a very short part at the top to say that this is a guest post and the Outdex would be happy to publish more of those.

rawskijon commented:
Alright, I fixed those up, and some other nitpicks I noticed. I updated the date for Sunday, but whenever you'd like to merge is fine. And add any preface you'd like! Thanks for bearing with me as I navigate this process *insert quote on benefits of active learning*
